03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+DIR="$(pwd)"
+FILE="file.txt"
+
+if [ -n "$1" ]; then
+	DIR="$1"
+fi
+
+echo "Search in $DIR"
+find "$DIR" -maxdepth 1 -type f -printf "size:%s\t" -exec md5sum {} \; > "$FILE"
```

Comment:
Is it possible to exclude temporary file usage?

Reply:
Done